src/RNCamera.js

```diff
@@ -322,6 +322,22 @@ export default class Camera extends React.Component<PropsType, StateType> {
     return await CameraManager.getAvailablePictureSizes(this.props.ratio, this._cameraHandle);
   };
 
+  async refreshAuthorizationStatus() {
```

Comment:
can you please change `componentDidMount` to call this method? It would be better to not have duplicated code.